packages/ui/stories/molecules/Increment.stories.tsx

```diff
@@ -60,6 +60,12 @@ storiesOf('ui/molecules/Increment', module)
           },
           defaultValue: {},
         },
+        isBigButton: {
+          control: {
+            type: 'boolean',
+          },
+          defaultValue: false,
+        },
```

Comment:
I asked you to update stories for all new props : readonly, defaultFormatting, stepSize, minValue, maxValue, isBigButton